docs/DesignDoc.md

```diff
@@ -32,16 +32,69 @@ This section describes the features of the application.
 
 ## Architecture and Design
 
-This section describes the application architecture.
-
 ### Software Architecture
-> _Place a architectural diagram here._
-> _Describe your software architecture._
+![architectural-pattern](..%2Fassets%2Farchitectural-pattern.png)
+
+For the software architecture design diagram of “TigerBets”, we adopt a layered approach. From top to bottom, it contains four layers:
+
+**Presentation Layer**:
+
+- Contains the User Interface (UI) and Simple Visual Charts.
+
+**Application Layer**:
+
+- Consists of Business Logic, Application Controllers for Authentication (Auth), Events, Accounts, Betting, and Analytics.
```

Comment:
Instead of bullet points under each layer, can be bring them to the same line since there is just one bullet under all the layers?

Example:
"**Presentation Layer**: Contains...."